src/Mod/Path/PathScripts/PathPocketBaseGui.py

```diff
@@ -22,50 +22,91 @@
+    FeatureFacing  ... used for face milling operation
+    FeatureOutline ... used for pocket-shape operation
+    """
 
     def pocketFeatures(self):
-        '''pocketFeatures() ... return which features of the UI are supported by the operation.
+        """pocketFeatures() ... return which features of the UI are supported by the operation.
           FeaturePocket  ... used for pocketing operation
           FeatureFacing  ... used for face milling operation
           FeatureOutline ... used for pocket-shape operation
-        Must be overwritten by subclasses'''
-        pass # pylint: disable=unnecessary-pass
+        Must be overwritten by subclasses"""
+        pass  # pylint: disable=unnecessary-pass
 
     def getForm(self):
-        '''getForm() ... returns UI, adapted to the results from pocketFeatures()'''
+        """getForm() ... returns UI, adapted to the results from pocketFeatures()"""
         form = FreeCADGui.PySideUic.loadUi(":/panels/PageOpPocketFullEdit.ui")
 
+        form.boundaryShape.clear()
+        form.boundaryShape.addItem(
+            QtCore.QT_TRANSLATE_NOOP("Path", "Boundbox"), "Boundbox"
+        )
+        form.boundaryShape.addItem(
+            QtCore.QT_TRANSLATE_NOOP("Path", "Face Region"), "Face Region"
+        )
+        form.boundaryShape.addItem(
+            QtCore.QT_TRANSLATE_NOOP("Path", "Perimeter"), "Perimeter"
+        )
+        form.boundaryShape.addItem(QtCore.QT_TRANSLATE_NOOP("Path", "Stock"), "Stock")
+
+        form.cutMode.clear()
+        form.cutMode.addItem(QtCore.QT_TRANSLATE_NOOP("Path", "Climb"), "Climb")
+        form.cutMode.addItem(
+            QtCore.QT_TRANSLATE_NOOP("Path", "Conventional"), "Conventional"
+        )
+
+        form.offsetPattern.clear()
+        form.offsetPattern.addItem(QtCore.QT_TRANSLATE_NOOP("Path", "ZigZag"), "ZigZag")
+        form.offsetPattern.addItem(QtCore.QT_TRANSLATE_NOOP("Path", "Offset"), "Offset")
+        form.offsetPattern.addItem(QtCore.QT_TRANSLATE_NOOP("Path", "Spiral"), "Spiral")
+        form.offsetPattern.addItem(
+            QtCore.QT_TRANSLATE_NOOP("Path", "ZigZagOffset"), "ZigZagOffset"
+        )
+        form.offsetPattern.addItem(QtCore.QT_TRANSLATE_NOOP("Path", "Line"), "Line")
+        form.offsetPattern.addItem(QtCore.QT_TRANSLATE_NOOP("Path", "Grid"), "Grid")
+        form.offsetPattern.addItem(
+            QtCore.QT_TRANSLATE_NOOP("Path", "Triangle"), "Triangle"
+        )
+
+        form.extraOffset.clear()
+
         if not FeatureFacing & self.pocketFeatures():
             form.facingWidget.hide()
             form.clearEdges.hide()
 
         if FeaturePocket & self.pocketFeatures():
             form.extraOffset_label.setText(translate("PathPocket", "Pass Extension"))
-            form.extraOffset.setToolTip(translate("PathPocket", "The distance the facing operation will extend beyond the boundary shape."))
+            form.extraOffset.setToolTip(
+                translate(
```

Comment:
If in this codebase this is a synonym for `QtCore.QT_TRANSLATE_NOOP` I suggest standardizing on one or the other. You'd probably save yourself some vertical space if you used the shorter one, if you're using Black.